# Warm temperatures, cool sponges: the effect of increased temperatures on the Antarctic sponge Isodictya sp.

### Marcelo González-Aravena, Nathan J. Kenny, Magdalena Osorio, Alejandro Font, Ana Riesgo and César A. Cárdenas

#### Reproduced by Mudith Ekanayake, Kavi Raj Acharya, Jaspreet Rishi

## Introduction
<p> <div style="text-align: justify"> Exposure to relatively high temperatures including acute thermal stress or heat shock has been studied extensively on cellular and molecular response but not much information is available on how it effects cold-water species.  In the Antarctic, estimates have shown that the temperature of seawater could rise by about 0.6°C to 2.0°C. This, particularly, could have effects on Antarctic-dwelling species as they might be vulnerable to these fluctuating conditions due to their specialization for extremely cold, relatively stable temperature conditions. 
In this study, a sponge, Isodictya sp, samples were collected from around the Doumer Island, Palmer Archipelago, western Antarctic Peninsula. 6 samples from adult Isodictya were collected from 10m deep to understand whether this organism can adjust to acute short-term heat stress, by exposing this species to 3°C and 5°C for four hours and the control group was maintained at 0.5°C (n=2). The temperatures corresponded to predicted temperatures under high-end 2080 IPCC-SRES (Intergovernmental Panel on Climate Change Special Report on Emissions Scenarios). 
After 4 hours of exposure, RNA was extracted from each sample and amplified using Illumina sequencing. Adapter sequences were removed, and overall quality of the sequences were tested using Fast QC and one low quality control was removed. Low- quality regions were trimmed using Trimmomatic 0.33. Clustering of isoforms was performed natively by Trinity. DeconSeq was used to remove contamination using pre-prepared bacterial, Archean and viral databases, resulting in the final reference assembly. 

## Technical Details

<p> <div style="text-align: justify"> Phylogenetic analysis and differential expression analysis were carried out in order to reproduce three figures from the paper; Figure 2A, Figure 2B and Figure 4. When reproducing the phylogenetic analysis, two different approaches were used; the approach which they have used in the paper and another approach using R. Command line blast (BLASTN) (Altschul et al., 1990) was used to extract the HSP70 sequences on HPC, using the sequences from Genbank as search queries. MAFFT sequence alignment tool (Katoh & Standley, 2013) was used to align the sequences and Gblocks (Castresana, 2000) was used to cure the alignments by removing spurious sequences or poorly aligned regions from the multiple sequence alignment. Phylogenetic reconstruction was undertaken under a maximum likelihood framework implemented in RAxML-NG (Silvestro & Michalak, 2012) using the LG+I+G model. Confidence levels were estimated using bootstrap resampling with 1000 replicates. Then the support values were mapped and the tree was visualized using itol tree visualizing tool (Letunik and Bork, 2019). All the annotations were added manually to make the tree look like the original figure in the paper (figure 4). Since it was not mentioned how they rooted their tree, reproduced tree was rooted just looking at the original figure and placing the root manually. </div> </p>

<p> <div style="text-align: justify"> In the other approach, phylogeny reconstruction was attempted to carry out in R. Aligning the sequences, improving the multiple sequence alignment quality and reconstructing the phylogeny were tried to perform using various R packages. But many issues were experienced and expected outcome was not possible. Since phylogeny was constructed using the first approach, many R packages were used to visualize, manipulate and root the tree in R. “ape” (Analysis of Phylogenetics and Evolution in R) was used mainly since it was the most important core package for phylogenies in R. Other packages used were phangorn, phytools and geiger. Tree was visualized as rectangular, radial, polar and also with node numbers in order to root the tree. Tree was rooted according to the node numbers since rooting method was not provided in the paper. </div> </p>

<p> <div style="text-align: justify"> Comparative analyses of gene exprassion was carried out using RSEM tool (Li & Dewey, 2011) as packaged within Trinity (Grabherr et al., 2011), using Bowtie2 (Langmead & Salzberg, 2012) for alignment. According to the paper the treatments were cross compared using edgeR (Robinson, McCarthy & Smyth, 2010) with several parameters; classic pairwise model with a p-value cut off for FDR of 0.0001, a minimum absolute change of 2, and a dispersion setting of 0.1. However, in this differential analysis step, there were some missing pieces and some parameters were not given. Also many supporting data had been used in the analysis, but they were not clearly mentioned in the paper. Because of that, in the middle of the analysis, many issues were experienced. Since some of the intermediate results as well as final expression results were included in the supplementary material, it was possible to incorporate those data when generating the heatmaps. Data preparation for reproducing the heatmaps were carried out in R using many R packages and heatmap generation was performed using the `heatmap.2()` function in `gplot` package. `heatmap.2()` is an enhanced and improved version of the basic `heatmap() function and it provides more options for generating more detailed heatmaps. </div> </p>

